src/intelli/data/stringlist.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * GTlab IntelliGraph
+ *
+ *  SPDX-License-Identifier: BSD-3-Clause
+ *  SPDX-FileCopyrightText: 2024 German Aerospace Center
+ *
+ *  Author: Jens Schmeink <jens.schmeink@dlr.de>
+ */
+
+#include "intelli/data/stringlist.h"
+
+using namespace intelli;
+
+StringListData::StringListData(QStringList val) :
+    NodeData(QStringLiteral("stringlist")),
+    m_data(std::move(val))
+{
+
+}
+
+QStringList
+intelli::StringListData::value() const
+{
+    return m_data;
+}
+
+void
+intelli::StringListData::setValue(const QStringList& val)
+{
+    m_data = val;
```

Comment:
Qt Containers are copy friendly, but still I like to use references and move semantics. Since we store the value we can return by reference and move it when assigning. 
```suggestion
QStringList const&
intelli::StringListData::value() const
{
    return m_data;
}

void
intelli::StringListData::setValue(QStringList val)
{
    m_data = std::move(val);
```